ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -109,7 +109,7 @@
                 <name-part name="family" font-variant="small-caps"/>
               </name>
               <et-al font-style="italic"/>
-              <label form="short" prefix="&#160;(" suffix=".)"/>
+              <label form="short" prefix=" (" suffix=".)"/>
```

Comment:
@ahahl, are you unescaping these non-breaking spaces on purpose (or replacing them by regular spaces)?